10-htcondor.conf

```diff
@@ -2,3 +2,6 @@
 command=/usr/sbin/condor_master -f
 autorestart=true
 startsecs=20
+stdout_logfile=/dev/fd/1
```

Comment:
```suggestion
stdout_logfile=/dev/stdout
```

Reply:
I'm not sure that this will actually provide anything useful since HTCondor dumps its own logs

Reply:
I think the suggestion and the original line are equivalent but yeah I think the whole block doesn't really help because it would only get us the master log.